netcheck/netcheck.go

```diff
@@ -454,11 +458,66 @@ func (c *Client) GetReport(ctx context.Context) (*Report, error) {
 		}
 	}
 
-	// TODO: if UDP is blocked, try to measure TCP connect times
-	// to DERP nodes instead? So UDP-blocked users still get a
-	// decent DERP node, rather than being randomly assigned to
-	// the other side of the planet? Or try ICMP? (likely also
-	// blocked?)
+	// Try HTTPS if UDP latency check failed for every stun server
+	// TODO: It does not show that UDP latency check failed
+	client := http.DefaultClient
+	client.Timeout = time.Second * 5
+	var rmu sync.Mutex
```

Comment:
it's not clear what this guards.

use style like:

```go
var (
     rmu sync.Mutex
     foo int
     bar string
)
```

... to make it obvious.